tracer/test/snapshots/MongoDbTests.SubmitsTraces_packageVersion=PRE_2_2.SchemaV0.verified.txt

```diff
@@ -7,11 +7,14 @@
     Service: Samples.MongoDB,
     Tags: {
       env: integration_tests,
+      language: dotnet,
       runtime-id: Guid_1,
-      version: 1.0.0
```

Comment:
This appears to be similar to my StackExchange update...we don't actually test this snapshot in automated testing, but I wanted to exercise all snapshots. So since the last time this file was updated, we had also added the language tag to the root span. Sorry for the confusion here 😅